core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -463,6 +463,12 @@ class TimeSeriesShard(val ref: DatasetRef,
 
   private[memstore] val addPartitionsDisabled = AtomicBoolean(false)
 
+  private[memstore] val housekeepingSched = Scheduler.computation(
```

Comment:
I'm just mirroring the DS setup-- unsure yet if this is the right way to go about things.